python/ModelTools.py

```diff
@@ -456,6 +457,7 @@ def doNuisances(self):
                         self.out.var(n).setRange(self.out.function('%s_BoundLo' % n), self.out.function('%s_BoundHi' % n))
                 else:
                     if len(args) == 3: # mean, sigma, range
+		    	sigma = double(args[2])
```

Comment:
just noticed that now that it should be `float` here (python has no `double`).
will include the fix in #576